WordPress/WordPressTest/ReferrerDetailsViewModelTests.swift

```diff
@@ -32,30 +34,27 @@ class ReferrerDetailsViewModelTests: XCTestCase {
         XCTAssertTrue(sut.isLoading)
     }
 
-    func testTableViewModel() {
-        XCTAssertEqual(sut.tableViewModel.sections.count, 1)
-
-        let firstChildRow = StatsTotalRowData(name: "Child 1", data: "Child data 1", disclosureURL: URL(string: "https://www.firstchild.com"))
-        let secondChildRow = StatsTotalRowData(name: "Child 2", data: "Child data 2", disclosureURL: URL(string: "https://www.secondchild.com"))
-        let data = StatsTotalRowData(name: "parent", data: "Data", disclosureURL: URL(string: "https://www.parent.com"), childRows: [firstChildRow, secondChildRow], isReferrerSpam: true)
-        sut.update(with: data)
-
+    func testTableViewModelOneDetailRow() {
```

Comment:
Thanks for splitting this up. 

I may be being nitpicky but I think the method names should be a bit more descriptive of what is being tested so at a glance one can determine what's happening and what is expected. 

A good example of this is [ReaderDetailCoordinatorTests.swift](https://github.com/wordpress-mobile/WordPress-iOS/blob/e767f0d41f479f2b799e489b83893da677e21a35/WordPress/WordPressTest/ReaderDetailCoordinatorTests.swift). 

What do you think?

Reply:
I agree, as discussed on Slack, I updated the code.